src/pyOpenMS/pxds/AASequence.pxd

```diff
@@ -136,7 +137,7 @@ cdef extern from "<OpenMS/CHEMISTRY/AASequence.h>" namespace "OpenMS":
 cdef extern from "<OpenMS/CHEMISTRY/AASequence.h>" namespace "OpenMS::AASequence":
         
         # static members
-        AASequence fromString(String s, bool permissive) except + nogil   # wrap-attach:AASequence wrap-as:fromStringPermissive
+        AASequence fromString(String s, bool permissive) except + nogil   # wrap-attach:AASequence wrap-as:fromStringPermissive wrap-doc:deprecated. Use AASequence(String) instead.
```

Comment:
Should be: use AASequence(String, bool) instead.
Also you can ref to other functions.
And as mentioned it would be better to use the deprecated admonition of sphinx to highlight it in the docs.

Reply:
ok I added an issue fot that. hope we come along adding this